examples/index.json

```
@@ -45,7 +46,8 @@
                 "name": "Identifiers",
                 "url": "identifiers",
                 "verifyBuild": "true",
-                "verifyOutput": "true"
+                "verifyOutput": "true",
+                "disablePlayground": true
```

Works on the updated play ground.

```suggestion
                "verifyOutput": "true"
```